Binary files orig/websocket/src/.Makevars.ucrt.swp and patched/websocket/src/.Makevars.ucrt.swp differ
diff -Nru orig/websocket/src/Makevars.ucrt patched/websocket/src/Makevars.ucrt
--- orig/websocket/src/Makevars.ucrt	2024-07-22 18:58:18.000000000 +0200
+++ patched/websocket/src/Makevars.ucrt	2025-03-26 19:49:06.497485400 +0100
@@ -1,2 +1,9 @@
-CRT=-ucrt
-include Makevars.win
+PKG_CPPFLAGS = -I./lib -D_WEBSOCKETPP_CPP11_THREAD_
+
+ifeq (,$(shell pkg-config --version 2>/dev/null))
+  PKG_LIBS = -lssl -lcrypto -lz -lws2_32 -lgdi32 -lcrypt32
+else
+  PKG_CPPFLAGS += $(shell pkg-config --cflags openssl)
+  PKG_LIBS = $(shell pkg-config --libs openssl)
+endif
+
